images/conda/config/template.apko.yaml

```diff
@@ -2,6 +2,8 @@ contents:
   packages:
     - busybox
     - conda
+    - conda-base
+    - conda-wrapper
 
 accounts:
   groups:
```

Comment:
Do we still need to define the groups/users we run as root?

Reply:
I'm not 100% certain on that. I followed the structure we currently use for other images that run as root. We could try without

Reply:
Looks like we're good!